README.md

```diff
@@ -83,17 +76,17 @@ a pull request.
 | `h8mk` | 48×48 8-bit mask                        | Yes        |
 | `it32` | 128×128 24-bit icon                     | Yes        |
 | `t8mk` | 128×128 8-bit mask                      | Yes        |
-| `icp4` | 16x16 32-bit PNG/JP2 icon               | PNG only   |
-| `icp5` | 32x32 32-bit PNG/JP2 icon               | PNG only   |
-| `icp6` | 64x64 32-bit PNG/JP2 icon               | PNG only   |
-| `ic07` | 128x128 32-bit PNG/JP2 icon             | PNG only   |
-| `ic08` | 256×256 32-bit PNG/JP2 icon             | PNG only   |
-| `ic09` | 512×512 32-bit PNG/JP2 icon             | PNG only   |
-| `ic10` | 512x512@2x "retina" 32-bit PNG/JP2 icon | PNG only   |
-| `ic11` | 16x16@2x "retina" 32-bit PNG/JP2 icon   | PNG only   |
-| `ic12` | 32x32@2x "retina" 32-bit PNG/JP2 icon   | PNG only   |
-| `ic13` | 128x128@2x "retina" 32-bit PNG/JP2 icon | PNG only   |
-| `ic14` | 256x256@2x "retina" 32-bit PNG/JP2 icon | PNG only   |
+| `icp4` | 16x16 32-bit PNG/JP2 icon               | Yes        |
```

Comment:
One thing not captured in this updated table is that JP2 encoding still isn't supported.  Maybe we could split the "Supported?" column into "Decoding" and "Encoding" columns, with all the PNG/JP2 rows set to "Yes" for "Decoding" but "PNG only" for "Encoding".

Reply:
Yeah, I did notice that as well, I think I also need to add that indication in the lib comments.